extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -434,6 +437,12 @@ private void handleHibernateORMWithNoPersistenceXml(
                             String resourceAsString = root.getArchiveRoot().relativize(loadScriptPath.get()).toString();
                             resourceProducer.produce(new SubstrateResourceBuildItem(resourceAsString));
                             desc.getProperties().setProperty(AvailableSettings.HBM2DDL_IMPORT_FILES, importFile);
+                            desc.getProperties().setProperty(AvailableSettings.HBM2DDL_IMPORT_FILES_SQL_EXTRACTOR,
+                                    MultipleLinesSqlCommandExtractor.class.getName());
+                            reflectiveClass.produce(
+                                    new ReflectiveClassBuildItem(false, false, MultipleLinesSqlCommandExtractor.class));
+                            reflectiveClass.produce(
+                                    new ReflectiveClassBuildItem(false, false, CommonToken.class));
```

Comment:
These 2 reflection registrations should be moved to `HibernateOrmReflections` (and thus the `reflectiveClass` parameter removed).